src/axom/core/Array.hpp

```diff
@@ -485,270 +477,263 @@ class Array
   double m_resize_ratio;
   bool const m_is_external;
 
-  DISABLE_COPY_AND_ASSIGNMENT( Array );
-  DISABLE_MOVE_AND_ASSIGNMENT( Array );
+  DISABLE_COPY_AND_ASSIGNMENT(Array);
+  DISABLE_MOVE_AND_ASSIGNMENT(Array);
 };
 
-
 //------------------------------------------------------------------------------
 //                            Array IMPLEMENTATION
 //------------------------------------------------------------------------------
 
 //------------------------------------------------------------------------------
-template< typename T >
-Array< T >::Array( ) :
-  m_data( nullptr ),
-  m_num_tuples( 0 ),
-  m_capacity( 0 ),
-  m_num_components( 1 ),
-  m_resize_ratio( DEFAULT_RESIZE_RATIO ),
-  m_is_external( false )
-{}
+template <typename T>
+Array<T>::Array()
+  : m_data(nullptr)
+  , m_num_tuples(0)
```

Comment:
Uncrustify was inconsistent about when it added/omitted spaces in parameters, conditionals and template brackets. I've set this to `false` (especially due to our 80 column limit), but there are options in clang-format if we vote to keep the spaces.